lpd/tests/test_models.py

```diff
@@ -199,15 +205,41 @@ def test_type(self):
         self.assertEqual(essay_question.type, QuestionTypes.ESSAY)
         self.assertEqual(short_answer_question.type, QuestionTypes.SHORT_ANSWER)
 
-    def test_get_answer(self):
+    @ddt.data(False, True)
+    def test_get_answer(self, split_answer):
         """
         Test that `get_answer` method returns appropriate value.
         """
-        question = QualitativeQuestionFactory()
+        question = QualitativeQuestionFactory(split_answer=split_answer)
         learner = UserFactory()
+
+        # Learner has yet to answer question
         self.assertEqual(question.get_answer(learner), '')
-        answer = QualitativeAnswerFactory(learner=learner, question=question, text='This is not an answer.')
-        self.assertEqual(question.get_answer(learner), answer.text)
+
+        answer_text = 'This, is, not, an, answer'
+        if split_answer:
+            for answer_component in answer_text.split(', '):
+                QualitativeAnswerFactory(learner=learner, question=question, text=answer_component)
+        else:
+            QualitativeAnswerFactory(learner=learner, question=question, text=answer_text)
+
+        # Learner answered question
+        self.assertEqual(question.get_answer(learner), answer_text)
+
+    @ddt.data(
+        (False, ['This, is, not, an, answer']),
+        (True, ['This', 'is', 'not', 'an', 'answer']),
+    )
+    @ddt.unpack
+    def test_get_answer_components(self, split_answer, expected_answer_components):
```

Comment:
@itsjeyd If we want to support answers with commas not followed by space (see my comment above), you might want to add change a bit `answer_text` and `expected_answer_components` to test it.
E.g. `This, is,not,an, answer`; `['This, is,not,an, answer']`